`#include <cpr/cpr.h>` would throw Error `No such file or directory` unless in `CMakeLists.txt`, `cpr` is changed to `cpr::cpr`

First of all, thank you for sharing the resourses of CS106L! I am not Stanford student so this issue might just be my environment. I am sharing it so that other people might find it helpful.

**Sumary**
A include error would be thrown if I just set up the project and build it. I found that in my environment, I need to change the 
 line `target_link_libraries(main PRIVATE cpr)` to `target_link_libraries(main PRIVATE cpr::cpr)`  in `CMakeLists.txt` so that the cpr can be properly included.

**Version and Platform**
`gcc 11.1.0`
`cmake 3.21.3`
And I am using Manjaro as OS.

**Procedure to replicate**
1. `git clone https://github.com/snme/cs106L-assignment1.git`
2. `cd cs106L-assignment1`
3. `./setup.sh`
4. (I actually did a bit modification of main.cpp, but I believe this is not related to this issue) `./build_and_run.sh`

the Error message is: 
```bash
cs106L-assignment1/WikiRacer/wikiscraper.h:4:10: fatal error: cpr/cpr.h: No such file or directory
    4 | #include <cpr/cpr.h>
      |          ^~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/main.dir/build.make:76: CMakeFiles/main.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/main.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
```
After I changed the line 12 in `CMakeLists.txt` `target_link_libraries(main PRIVATE cpr)` to `target_link_libraries(main PRIVATE cpr::cpr)`, the build runs successfully. I am not 100% sure why that is the case, but maybe it's just a version issue? This change is suggested by the installation message when I run `./vcpkg install cpr` separately.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -9,4 +9,4 @@ find_package(cpr CONFIG REQUIRED)
 # adding all files
 add_executable(main main.cpp wikiscraper.cpp.o error.cpp)
 
-target_link_libraries(main PRIVATE cpr)
+target_link_libraries(main PRIVATE cpr::cpr)

